index.html

```diff
@@ -1,247 +1,368 @@
+                  <li>html</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button class="btn-desk btn btn-card" type="button">
+                See Project
+              </button>
+            </li>
+            <li class="project-card">
+              <div class="Card-description">
+                <h3>Profesional Art Printing Data</h3>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="Description-tags">
+                  <li>html</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button class="btn-desk btn btn-card" type="button">
+                See Project
+              </button>
+            </li>
+            <li class="project-card">
+              <div class="Card-description">
+                <h3>Profesional Art Printing Data</h3>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="Description-tags">
+                  <li>html</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button class="btn-desk btn btn-card" type="button">
+                See Project
+              </button>
+            </li>
+            <li class="project-card">
+              <div class="Card-description">
+                <h3>Profesional Art Printing Data</h3>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="Description-tags">
+                  <li>html</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button class="btn-desk btn btn-card" type="button">
+                See Project
+              </button>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <div id="About-and-skill">
+        <section id="about-section">
+          <div class="Header">
+            <h3 id="About-me">About me</h3>
+          </div>
+          <p class="About-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button type="button" class="btn btn-About-me">Get My Resume</button>
+        </section>
```

Comment:
- [ ] kindly the background image is very big fix it and increase the size of the `Header` to make it visible. 
Your Design
![Screenshot from 2022-06-03 14-28-46](https://user-images.githubusercontent.com/77842237/171846671-eb1ef70e-a168-459d-b3ae-93b53b0eb2de.png)

Figma Design
![Screenshot from 2022-06-03 14-42-10](https://user-images.githubusercontent.com/77842237/171847080-8ee1af37-f787-496c-8f69-427213130240.png)

Reply:
Changes required has been corrected.